CHANGELOG.md

```diff
@@ -58,6 +62,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add support for `verify_email_by_code` email template [#309]
 
+## [5.3.2] - 2020-12-17
+### Added
+- Adds support for Actions.
```

Comment:
🤔  should this be here? I thought actions was only in 6.x

Reply:
🤔  why was that added